Tools/TestWebKitAPI/TestBundle/TestWebKitAPI.xctestplan

```diff
@@ -14,7 +14,7 @@
   "testTargets" : [
     {
       "target" : {
-        "containerPath" : "container:TestWebKitAPI.xcodeproj",
+        "containerPath" : "container:..\/OpenSource\/Tools\/TestWebKitAPI\/TestWebKitAPI.xcodeproj",
```

Comment:
It's a bit surprising to see a relative path like this in open source code. I thought the OpenSource/Internal distinction only makes sense in the context of compiling WebKit/Safari using the Apple internal SDK?

Reply:
+1 this doesn't seem right